pkg/minishift/addon/command/command.go

```diff
@@ -31,12 +31,16 @@ type doExecute func(ec *ExecutionContext) error
 type defaultCommand struct {
 	Command
 
-	rawCommand string
-	fn         doExecute
+	rawCommand  string
+	fn          doExecute
+	ignoreError bool
 }
 
 func (c *defaultCommand) Execute(ec *ExecutionContext) error {
 	err := c.fn(ec)
+	if c.ignoreError {
+		return nil
```

Comment:
this is the whole change... ;-)
the rest is related to passing the variable and tests

Reply:
:) ya